src/Entity/CitizenProject.php

```diff
@@ -672,4 +672,16 @@ public function setNextAction(CitizenAction $citizenAction): void
     {
         $this->nextAction = $citizenAction;
     }
+
+    public function exportCommitteeSupports(): string
+    {
+        return join(', ', array_map(function (CitizenProjectCommitteeSupport $committeeSupport) {
+            return $committeeSupport->getCommittee()->getName();
```

Comment:
It would be nice to know the status of the committee on the exported sheet to avoid confusion.
Either we should only export approved supports, either we should display the status of each support.